docs/Developers/Review.rst

```diff
@@ -3,19 +3,51 @@ Reviewing
 
 This document is a guide to coala's review process.
 
+Am I Good Enough to Do Code Review?
+-----------------------------------
+
+**Yes,** if you already fixed a newcomer issue.
```

Comment:
One thing I'm generally missing is how helpful it can be for newcomers to engage in reviewing. They are not only doing something for us, they are also helping themselves. They learn about how to improve their own commits, different internals of coala, different people, how to code better in general, ...